ArkKit/ark-event-kit/ArkEvent.swift

```diff
@@ -7,12 +7,18 @@
 
 import Foundation
 
-typealias ArkEventID = UUID
+typealias ArkEventID = String
 
-protocol ArkEvent<Data> {
+protocol ArkEvent<Data>: Codable {
     associatedtype Data = ArkEventData
 
-    static var id: ArkEventID { get }
+    static var id: String { get }
```

Comment:
Should this be ArkEventID? if not line 10 is redundant

Reply:
Removed `id`